sdk/openai/openai/samples/cookbook/simpleCompletionsPage/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "azure-openai-browser-completions",
+  "version": "1.0.0",
+  "description": "SPA to demonstrate browser completions using Azure OpenAI",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --mode=production --node-env=production",
+    "build:dev": "webpack --mode=development",
+    "build:prod": "webpack --mode=production --node-env=production",
+    "watch": "webpack --watch"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@azure/openai": "^1.0.0-beta.2",
+    "child_process": "^1.0.2"
+  },
+  "devDependencies": {
+    "@webpack-cli/generators": "^3.0.4",
+    "babel-loader": "^9.1.2",
+    "html-webpack-plugin": "^5.5.1",
+    "webpack": "^5.86.0",
+    "webpack-cli": "^5.1.4"
+  },
+  "browser":{
+    "child_process": false
```

Comment:
is `child_process` used? I don't fully understand adding the dependency but then setting it to false for browser.